packages/cli/src/Server.ts

```diff
@@ -1696,8 +1696,10 @@ class App {
 					);
 				}
 
-				const encryptionKey = await UserSettings.getEncryptionKey();
-				if (!encryptionKey) {
+				let encryptionKey: string;
+				try {
+					encryptionKey = await UserSettings.getEncryptionKey();
+				} catch (error) {
```

Comment:
ref comment in `UserSettings.ts`

Reply:
IN this case specifically we're changing the regular error to a new type of error, so this try / catch block is still necessary